### DISCUSSION TITLE
How to do this please?

### DISCUSSION THREAD

**Discussion by MiroslavFPV** [EdgeTX/edgetx#1108] [Q&A] [answered]
Hello, i set my trims to multipos switch to change channel and band in vtx by Bardwells video (https://youtu.be/wa43z6XohW4) but i want one more function and dont know how to do it. I want something like:

If:                                  
Input is enabled (momentary switch) 

Do:
Disable special function x. (so mix cant move)      

I want it for something like fuse that i dont change my channel by accidentaly pushing trim. Is it possible? Thanks for replyes.

**Comment by pfeerick** [comment]
I'll have a look and play, but I suspect you'll need to approach it the other way, and each button should check for the state of that 'enable' switch.

**Comment by MiroslavFPV** [comment, marked as answer]
Thanks for reply, i did it by AND logical.

Dňa ne 21. 11. 2021, 1:57 Peter Feerick ***@***.***>
napísal(a):
